assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties

```diff
@@ -253,288 +253,277 @@ db.schema.flyway.scripts.prefix=
-# If false, an administrator is required to label the namespaces manually using the labels specified in che.infra.kubernetes.namespace.labels.
-# If you want to manage the namespaces yourself, make sure to follow
+# Defines whether {prod-short} server should try to label the workspace namespaces.
+# NOTE: It's strongly recommended you keep the value of this property set to `true`. If set to `false`, the new workspace namespaces isn't labeled
+# automatically, and therefore not recognized by the {prod-operator-image-name} causing some features of DevWorkspaces to not work.
+# If set to `false`, an administrator is required to label the namespaces manually using the labels specified in `che.infra.kubernetes.namespace.labels`.
+# If you want to manage the namespaces yourself, make sure to follow this guide:
 # https://www.eclipse.org/che/docs/stable/administration-guide/provisioning-namespaces-in-advance/.
-# Any additional labels present on the namespace are kept in place and do not affect the functionality.
+# Any additional labels present on the namespace are kept in place and don't affect the functionality.
 # Also note that the the administrator is free to pre-create and label the namespaces manually even if this property is
-# true. No updates to the namespaces are done if they already conform to the labeling requirements.
+# `true`. No updates to the namespaces are done if they already conform to the labeling requirements.
 che.infra.kubernetes.namespace.label=true
 
-# Defines whether che-server should try to annotate the workspace namespaces.
+# Defines whether {prod-short} server should try to annotate the workspace namespaces.
 che.infra.kubernetes.namespace.annotate=true
 
 # List of labels to find {orch-namespace} that are used for {prod-short} Workspaces.
-# They are used to:
-#  - find prepared {orch-namespace} for users in combination with `che.infra.kubernetes.namespace.annotations`.
-#  - actively label {orch-namespace} with any workspace.
-# NOTE: It is strongly recommended not to change the value of this property because the Che operator relies on these labels 
-# and their precise values when reconciling DevWorkspaces. If this configuration is changed, the namespaces will not be automatically
-# recognized by the Che operator as workspace namespaces unless manually labeled as such using the default labels and values.
-# Additional labels on the namespace do not affect the functionality.
+# They perform the following actions:
+#  - Find prepared {orch-namespace} for users in combination with `che.infra.kubernetes.namespace.annotations`.
+#  - Actively label {orch-namespace} with any workspace.
+# NOTE: It's strongly recommended not to change the value of this property because the {prod-operator-image-name} relies on these labels 
+# and their precise values when reconciling DevWorkspaces. If this configuration is changed, the namespaces isn't automatically
```

Comment:
If... will be.

Reply:
also you switched to a verb form that's singluar for a plural noun.

`the namespaces aren't` would be correct, if this wasn't a conditional sentence. 

Please revert to "If this configuration is changed, the namespaces will not be automatically"